Language/.repo-metadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "distribution_name": "google/cloud-language",
-    "release_level": "beta",
+    "release_level": "ga",
```

Comment:
This is an ML library that still has a handwritten, would we like to remove the handwritten before doing this?

Reply:
Agreed that we should remove the handwrittens! I'll do this in a separate PR